### PR TITLE
Create middleware.js

### PR DESCRIPTION
**Pull request by yn1018**
add middleware

### CODE REVIEW

File: middleware.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log('middleware');
```
Comment (SmartRenaldo) [inline middleware.js:1]: maybe be more specific for this middleware

**Review by SmartRenaldo — CHANGES_REQUESTED** [review]
Just a few changes, then good to go.

File: middleware.js (hunk 1 above)
Comment (yn1018) [reply middleware.js:1]: Yep, will fix.

File: middleware.js [hunk 2]
```diff
@@ -1 +1,5 @@
 console.log('middleware');
+let x = 5;
+let y = 2;
```
Comment (SmartRenaldo) [inline middleware.js:4]:
```suggestion
add some comments
```

**Review by SmartRenaldo — CHANGES_REQUESTED** [review]
All good, just add some comments

File: middleware.js (hunk 2 above)
Comment (yn1018) [reply middleware.js:4]: will fix it. Thx

**Review by SmartRenaldo — APPROVED** [review]
Good job

**Pull request merged**